core/tests/data/test_cases_for_rte.json

```diff
@@ -0,0 +1,275 @@
+{"RTE_TYPE_TEXTANGULAR": {
+  "TEST_CASES": [{
```

Comment:
@bansalnitish, Can you please check whether we still need this file? 
[Abhith was working on some RTE stuff and found this file and we aren't sure whether we still need this file.]

Reply:
I searched around the codebase and I don't think we need this file anymore. Thanks!

Reply:
Thanks, @bansalnitish for the update.

Reply:
You're welcome!